libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/iterator/increment.pass.cpp

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// REQUIRES: libcpp-hardening-mode={{fast|extensive|debug}}
```

Comment:
You could also say this instead, which might be more forward-proof with adding a new hardening mode:

```suggestion
// UNSUPPORTED: libcpp-hardening-mode=none
```